src/clj/villagebook/organisation/models.clj

```diff
@@ -0,0 +1,18 @@
+(ns villagebook.organisation.models
+  (:require [villagebook.organisation.db :as db]
+            [villagebook.factory :as factory]))
+
+(defonce DEFAULT_PERMISSION :owner)
+
+(defn create!
```

Comment:
The fn works under the assumption that db calls will always work. It's worth wrapping this in a try-catch block and returning some error map for failed cases.

Reply:
Would it not be better in this case to let the exception bubble up to the handler? Ring will send a 500 in that case since it _is_ an internal exception. There might be too many cases to acknowledge explicitly.

Reply:
But the default exception message may or may not be useful, wrapping it here gives a chance for better formatted error strings.